### PR TITLE
Feature/footer

### PR DESCRIPTION
**Pull request by meghz-glitch**
I added new footer.css

### CODE REVIEW

File: styles/footer.css [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+  }
+  
+  .footer-content {
+    display: flex;
+    justify-content: space-between;
+    flex-wrap: wrap;
+    max-width: 1200px;
+    margin: auto;
+  }
+
```
Comment (Svpriyaa2808) [inline styles/footer.css:17]: remove whitespaces

File: styles/footer.css [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+  .footer-bottom ul li {
+    margin: 0 10px;
+  }
+  
+  .footer-bottom ul li a {
+    color: #333;
+    text-decoration: none;
+  }
+  
+  .footer-bottom ul li a:hover {
```
Comment (Svpriyaa2808) [inline styles/footer.css:104]: is media query needed or it looks responsive

File: styles/footer.css [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+    margin-right: 10px;
+    font-size: 24px;
+    transition: color 0.3s ease;
+  }
+  
+  .social-icon:hover {
+    color: #4267B2; 
+  }
+  
+
```
Comment (Kavitha0691) [inline styles/footer.css:78]: remove white line

File: index.html [hunk 4]
```diff
@@ -32,8 +34,49 @@
...
+<div class="footer-left">
+  <h2>Nature Zoo</h2>
+  <p>
+        Nature Zoo is a 700-acre (280 ha) wildlife sanctuary tucked beneath Queensland’s mystical Beerwah and Glass House Mountains. This magical haven, owned by Terri Irwin, continues the adventurous spirit of her late husband, Steve Irwin, whose legacy lives on through the family’s show, Mysteries of the Wild.
+        Founded in 1970 by Bob and Lyn Irwin as Beerwah Reptile and Fauna Park, Nature Zoo has grown into a vibrant realm where kangaroos roam freely, crocodiles reign, and every visit feels like stepping into a living fairy tale.
+  </p>
+  <a href="#" class="newsletter-link">Join our newsletter →</a>
+</div>
+
+
```
Comment (Kavitha0691) [inline index.html:49]: remove white line here

File: index.html [hunk 5]
```diff
@@ -32,8 +34,49 @@
...
     </div>
-    <footer>
-      <p>&copy; 2024 Australia Zoo Team. All rights reserved.</p>
-    </footer>
+   <!-- Footer -->
+<footer>
+  <div class="footer-content">
+<div class="footer-left">
+  <h2>Nature Zoo</h2>
+  <p>
```
Comment (Kavitha0691) [inline index.html:42]: add all these through JS

File: index.html [hunk 6]
```diff
@@ -32,8 +34,49 @@
...
+    <a href="https://x.com/" target="_blank" class="social-icon">
+      <i class="fab fa-x"></i>
+    </a>
+    <a href="https://www.instagram.com/" target="_blank" class="social-icon">
+      <i class="fab fa-instagram"></i>
+    </a>
+  </div>
+</div>
+</div>
+
```
Comment (Kavitha0691) [inline index.html:71]: remove empty line

File: styles/footer.css [hunk 7]
```diff
@@ -0,0 +1,106 @@
...
+  .footer-left p {
+    line-height: 1.6;
+    margin-bottom: 20px;
+  }
+  
+  .newsletter-link {
+    color: #007bff;
+    text-decoration: none;
+  }
+
```
Comment (Kavitha0691) [inline styles/footer.css:37]: remove empty line

File: index.html [hunk 8]
```diff
@@ -32,8 +34,49 @@
...
+        Nature Zoo is a 700-acre (280 ha) wildlife sanctuary tucked beneath Queensland’s mystical Beerwah and Glass House Mountains. This magical haven, owned by Terri Irwin, continues the adventurous spirit of her late husband, Steve Irwin, whose legacy lives on through the family’s show, Mysteries of the Wild.
+        Founded in 1970 by Bob and Lyn Irwin as Beerwah Reptile and Fauna Park, Nature Zoo has grown into a vibrant realm where kangaroos roam freely, crocodiles reign, and every visit feels like stepping into a living fairy tale.
+  </p>
+  <a href="#" class="newsletter-link">Join our newsletter →</a>
+</div>
+
+
+<div class="footer-right">
+  <ul>
+    <li><a href="#">About the Zoo</a></li>
```
Comment (Kavitha0691) [inline index.html:52]: add all these through JS

**Review by Kavitha0691 — COMMENTED** [review]
reviewed..
add JS for this

**Review by Kavitha0691 — APPROVED** [review]
reviewd

**Pull request merged**